packages/@aws-cdk/aws-codedeploy/package.json

```diff
@@ -41,7 +41,7 @@
   },
   "nyc": {
     "lines": 50,
-    "branches": 40
+    "branches": 30
```

Comment:
Why?

Reply:
Because there wasn't an easy way of asserting on the User Data in the tests, and I'd rather not copy&paste the entire script in the tests.

Reply:
Sorry but that’s not an excuse not to test something. If you need to maintain a copy as an expected file to ensure that your code works, then that’s what you should do.